app/serializers/user_serializer.rb

```diff
@@ -24,4 +24,13 @@ class UserSerializer
       }
     end
   end
+
+  attribute :address do |user|
+    {
+      street: user.street,
+      city: user.city,
+      state: user.state,
+      zipcode: user.zipcode
```

Comment:
You made good use of the previous address attribute and gave it keys for the user's address information; good work!